main/utils/api_exceptions.py

```diff
@@ -0,0 +1,33 @@
+"""
+Useful classes which can be used to specify the Error type when something failed.
+
+Classes:
+    RestError
+"""
+class RestError(AssertionError):
+    """Class which can be used to specify the Rest request has failed"""
+    def __init__(self, status_code, request, response):
+        """
+        :param status_code: int  Status code returned by the service through the API
+        :param request:     str  Full URL endpoint to which the request was sent
+        :param response:    str  Response which contains info about the failure 
```

Comment:
I'm confused by this parameter: `response` is declared to be a `str`, but in lines 24,25,28, and 33 it's accessed as if it were an Object, right?